tools/proteore_topgo/.shed.yml

```diff
@@ -0,0 +1,13 @@
+categories: [Proteomics]
+description: ProteoRE topGO
+long_description: A tool performing enrichment analysis for Gene Ontology
+name: proteore_topgo
+owner: proteore
+include:
```

Comment:
I don't think we need this `include` directive ... which files do you want exclude here?

Reply:
you are right. Is it possible to remove all the lines frim 'include' then ?

Reply:
yes, indeed we are not using this at all in our tools. You only need this if you ship files in github that you don't want to have in the ToolShed.